src/basic/parse-util.c

```diff
@@ -747,3 +747,34 @@ int parse_loadavg_fixed_point(const char *s, loadavg_t *ret) {
 
         return store_loadavg_fixed_point(i, f, ret);
 }
+
+static bool nft_first_char_good(const char c) {
+        return (c >= 'a' && c <= 'z') ||
+                (c >= 'A' && c <= 'Z');
+}
+
+static bool nft_next_char_good(const char c) {
+        return (c >= 'a' && c <= 'z') ||
+                (c >= 'A' && c <= 'Z') ||
+                (c >= '0' && c <= '9') ||
+                c == '/' || c == '\\' || c == '_' || c == '.';
+}
+
+/* Limitations are described in https://www.netfilter.org/projects/nftables/manpage.html and
+ * https://bugzilla.netfilter.org/show_bug.cgi?id=1175 */
+bool nft_identifier_good(const char *id) {
```

Comment:
we typically call functions that validate strings "xyz_is_valid()" or "xyz_valid()". please follow the same logic here.

Reply:
Now _valid